### DISCUSSION TITLE
receive updates for tags and categories

### DISCUSSION THREAD

**Discussion by pragyad3188** [vmware/govmomi#2880] [Community Help] [answered]
Hi,

I wanted to know if there is any way to get updates for Tags and Categories.

I know that for managed objects like VirtualMachine, VMware provides PropertyFilter and WaitForUpdates API to get updateset for the resources and their properties which are added in PropertyFilter. 
I've referred to the https://pkg.go.dev/github.com/vmware/govmomi/vapi/tags package, but couldn't find any API which can help in receiving updates for the Tags and Categories. 

Thank you in advance!

**Comment by embano1** [comment, marked as answer]
IMHO there's currently no VMODL2 (VAPI) API or event which reflects property changes on tags/categories. The only event related to tags we have is `com.vmware.cis.tagging.attach` and `com.vmware.cis.tagging.detach`.
